### ISSUE TITLE
How to listen for the 'submit' event or implement handleSubmit?

### ISSUE THREAD

**Issue by TaylorAckley** [kareniel/angularjs-stripe-elements#3]
Howdy - I have been unable to figure out how to finish using the library.

I can get the form to show up and listen to the change event.   The example shows a `handleSubmit` function being passed to `ng-submit` but I can't seem to set it up right.    I try passing arguments to it, but it doesn't really give me anything.

Reading the stripe documentation, it says I am supposed to set up a listener for the submit event, but that doesn't seem to "catch'.  https://stripe.com/docs/stripe-js/elements/quickstart

For example:

```
	element.on('submit', handleSubmit);
	function handleSubmit(e) {
		console.log(e);
	}
```

The event doesn't seem to show in the console.

Then doing this doesn't seem to yield the 'card' element.

```
	ctrl.handleSubmit = function($event) {
		console.log(ctrl.element);
	};
````

Any help would be greatly appreciated, thank you!

**Comment by kareniel** [comment]
Hi Taylor!

I'm not sure I have all the information necessary to be able to help you.
Is there any way you could share more of your app's code?

In the meantime, I suggest you check out the example folder in this repository and compare it with your code to make sure there's nothing missing. 

Let me know!

**Comment by TaylorAckley** [comment]
Thanks for responding on a Saturday!   I finally got it figured out.

I checked the example code but couldn't figure out how to implement the `handleSubmit` function that is specified in `ng-submit`    The example code didn't contain a handleSubmit function so it took a little bit of guess work but I realized I could take the code from the Stripe page and I just needed to rename the element.  Final result was adding this function to your example code.

```
	ctrl.handleSubmit = function($event) {
		console.log(ctrl.product);
		StripeElements.createToken(element).then(function(result) {
			if (result.error) {
				ctrl.cardErrors = result.error.message;
			} else {
				// Send the token to your server.
				console.log(result);
			}
		});
	};
```
